tests/scripts/thread-cert/Cert_8_3_01_CommissionerPetition.py

```diff
@@ -357,7 +357,9 @@ def verify(self, pv):
                               NM_STATE_TLV,
                               NM_COMMISSIONER_SESSION_ID_TLV
                               } <= set(p.coap.tlv.type) and\
-                   p.thread_meshcop.tlv.state == MESHCOP_ACCEPT
+                   p.thread_meshcop.tlv.state == MESHCOP_ACCEPT and\
+                   (p.thread_meshcop.tlv.commissioner_sess_id -
+                   _dr_pkt3.thread_meshcop.tlv.commissioner_sess_id) % 256 <= 127
```

Comment:
One note is that Session ID is of type unit16_t with random initialization, and increment (+1) along per new active commissioner.
https://github.com/openthread/openthread/blob/9380ffd07c39d7a2cfe9bca8cf2c7649873e6da2/src/core/meshcop/meshcop_leader.cpp#L99